app/models/base_policy.rb

```diff
@@ -58,4 +58,12 @@ def further_education_payments?
   def auto_check_student_loan_plan_task?
     false
   end
+
+  def award_amount_column
+    if to_s == "StudentLoans"
```

Comment:
Perhaps the `StudentLoans` model should override this method instead

Reply:
Yeah that's nicer, have fixed 👍